capsules/src/spi.rs

```diff
@@ -248,28 +249,28 @@ impl<'a, S: SpiMaster> Driver for Spi<'a, S> {
             }
             9 /* set polarity */ => {
                 match arg1 {
-                    0 => self.spi_master.set_clock(ClockPolarity::IdleLow),
-                    _ => self.spi_master.set_clock(ClockPolarity::IdleHigh),
+                    0 => self.spi_master.set_polarity(ClockPolarity::IdleLow),
+                    _ => self.spi_master.set_polarity(ClockPolarity::IdleHigh),
                 };
                 0
             }
             10 /* get polarity */ => {
-                self.spi_master.get_clock() as isize
+                self.spi_master.get_polarity() as isize
             }
             11 /* hold low */ => {
-                self.spi_master.hold_low();
+//                self.spi_master.hold_low();
                 0
             }
             12 /* release low */ => {
-                self.spi_master.release_low();
+ //               self.spi_master.release_low();
```

Comment:
I assume these are commented out because it can't be supported virtualized without power locks?